syncopy/shared/computational_routine.py

```diff
@@ -1019,15 +1019,14 @@ def process_metadata(self, data, out):
 
 # --- metadata helper functions ---
 
-def propagate_metadata(in_data, out_data):
+def propagate_properties(in_data, out_data, keeptrials=True):
```

Comment:
very nice, this reduces quite a bit code duplication

Reply:
As discussed, it's still not perfect imho.. I could also think of putting those property mappings into the respective module `compRoutines.py`, however there is already quite some crossover between `freqanalysis` and `connectivityanalysis` for example.